tekton/README.md

```diff
@@ -155,49 +182,39 @@ access), such as [the production service account](#production-service-account).
 
 To run the `publish-tekton-pipelines` `Task` and create a release:
 
-1. Pick the revision you want to release and replace the value of the
-   `PipelineResource` [`tekton-pipelines` `revision`](publish-run.yaml#11),
-   e.g.:
+1. Pick the revision you want to release and update the
+   [`resources.yaml`](./resources.yaml) file to add a
+   `PipelineResoruce` for it, e.g.:
 
    ```yaml
-   - name: revision
-     value: 67efb48746a9d5d7d3b9b5c5cc210de7a47c6ebc # REPLACE with your own commit
+   apiVersion: tekton.dev/v1alpha1
+   kind: PipelineResource
+   metadata:
+     name: tekton-pipelines-v0-5-2
+   spec:
+     type: git
+     params:
+     - name: url
+       value: https://github.com/tektoncd/pipeline # REPLACE with your own fork
+     - name: revision
+       value: v0.5.2 # REPLACE with your own commit
```

Comment:
in the example id rather include a version that is clearly invalid just in case someone accidentally copies + pastes this (and overwrites an actual release 😅 #983)